.github/workflows/release.yml

```diff
@@ -40,7 +43,38 @@ jobs:
           aws-access-key-id: ${{ secrets.COREINT_AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.COREINT_AWS_SECRET_ACCESS_KEY }}
           aws-region: us-east-1
-      - name: Upload tarball and configuration files to https://download.newrelic.com
+      - name: Upload tarball to https://download.newrelic.com
         run: |
           make release_tarball_package_for OS=$OS ARCH=$ARCH RELEASE_VERSION=$RELEASE_VERSION
+
+  release-manifest:
```

Comment:
gha needs a sync.doOnce condition for the steps to avoid this :)

Reply:
what :O I'll run to change that then!